### PR TITLE
Added description of sensors created by the Tibber integration

### PR DESCRIPTION
**Pull request by Bejjje**
## Proposed change

Added description of sensors created by integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

A handfull of sensors created by the integration is suspected to be deprecated from Tibbers side and mentiond as such.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tibber.markdown [hunk 1]
```diff
@@ -83,4 +116,10 @@ The electricity price can be used to make automations. The sensor has a `max_pri
        message: "The electricity price is now {{ states('sensor.electricity_price_hamretunet_10') }}"
 ```
 
+### URL
+
+https://www.home-assistant.io/integrations/tibber/
+
```
Comment (frenck) [inline source/_integrations/tibber.markdown]: Why is this added?

File: source/_integrations/tibber.markdown [hunk 2]
```diff
@@ -56,6 +56,39 @@ You also get sensors for monthly consumption, monthly cost, and monthly peak hou
 If you have a Tibber Pulse it will also show the electricity consumption in real time.
 
 
+## Available Sensors
```
Comment (frenck) [inline source/_integrations/tibber.markdown:58]: Please don't list sensors by their entity IDs. They are not guaranteed and don't have to be the same on your system.

Looking at the naming use, it is based on an old naming scheme as well, so already incorrect.

Instead, please just describe the entities.

PS: Please avoid the use of tables. They are generally hard to read on mobile.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Oh, could you also update the PR title to be descriptive and fill out the PR template/description? It would be nice to elaborate/motivate the changes you are making.

File: source/_integrations/tibber.markdown (hunk 1 above)
Comment (Bejjje) [reply source/_integrations/tibber.markdown]: Cut and paste error on my behalf, removed.

File: source/_integrations/tibber.markdown (hunk 2 above)
Comment (Bejjje) [reply source/_integrations/tibber.markdown:58]: Adjusted in accordance (hopefully)

**Comment by Bejjje** [comment]
I seem to be in way over my head into Github (first time attempt). 
If I managed to get the changes done and produce something that seems to be of value, keep it.
Otherwise discard everything and my struggling attempts.

File: source/_integrations/tibber.markdown [hunk 3]
```diff
@@ -55,6 +55,37 @@ The `tibber` sensor provides the current electricity price if you are a [Tibber]
 You also get sensors for monthly consumption, monthly cost, and monthly peak hour
 If you have a Tibber Pulse it will also show the electricity consumption in real time.
 
+## Available Sensors
+
+- Accumulated cost since midnight (requires active Tibber power deal)
+- kWh consumed since midnight
+- net kWh produced since midnight
+- net kWh produced since last hour shift
+- Current on L1, L2 and L3
```
Comment (frenck) [inline source/_integrations/tibber.markdown:64]:
```suggestion
- Current on L1, L2, and L3
```

File: source/_integrations/tibber.markdown [hunk 4]
```diff
@@ -55,6 +55,37 @@ The `tibber` sensor provides the current electricity price if you are a [Tibber]
...
+- Estimate of kWh consumption current hour
+- kWh consumed since since last hour shift
```
Comment (frenck) [inline source/_integrations/tibber.markdown:66]:
```suggestion
- kWh consumed since last hour shift
```

File: source/_integrations/tibber.markdown [hunk 5]
```diff
@@ -55,6 +55,37 @@ The `tibber` sensor provides the current electricity price if you are a [Tibber]
...
+- Last meter active import register state (kWh)
+- Last meter active export register state (kWh)
+- Peak consumption since midnight (Watt)
+- Min consumption since midnight (Watt)
+- Consumption at the moment (Watt)
+- Consumption at the moment (Watt)
+- Net production (A-) at the moment (Watt)
+- The total price (energy + taxes)
+- Device signal strength (Pulse - dB; Watty - percent)
+- Voltage on phase 1, 2 and 3
```
Comment (frenck) [inline source/_integrations/tibber.markdown:77]:
```suggestion
- Voltage on phases 1, 2, and 3
```

File: source/_integrations/tibber.markdown [hunk 6]
```diff
@@ -55,6 +55,37 @@ The `tibber` sensor provides the current electricity price if you are a [Tibber]
...
+<div class='note'>
+
+Sensors for information below are created but suspected to be deprecated from Tibbers side since they aren't populated
+
+- Monthly cost
+- Monthly net consumption
+- Monthly peak hour
+- Time of max hour consumption
+
+</div>
```
Comment (frenck) [inline source/_integrations/tibber.markdown]: We should not add this unless we are sure. (and thus not suspecting).

In case they are deprecated, we should remove them from the codebase. A comment in the docs will not clean them up or fix an issue.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tibber.markdown (hunk 6 above)
Comment (Bejjje) [reply source/_integrations/tibber.markdown]: Removed. 
Now these sensors are being populated with data again so the suspicion was incorrect.

**Review by frenck — APPROVED** [review]
Thanks, @Bejjje 👍
